### PR TITLE
39 remove update from update routes

### LINKED ISSUE #39: remove "update" from update routes

### PR DESCRIPTION
**Pull request by ralphZero**
-  removed "update" from update routes

### CODE REVIEW

File: src/routes/newsRoutes.ts [hunk 1]
```diff
@@ -8,6 +8,6 @@ router.route('/').get(auth, getNews)
 router.route('/:newsId').get(auth, getNewsById)
 router.route('/').post(auth, addNews)
 router.route('/disable/:newsId').patch(auth, disableNews)
-router.route('/update/:newsId').patch(auth, updateNews)
+router.route('/:newsId').patch(auth, updateNews)
```
Comment (jonathans199) [inline src/routes/newsRoutes.ts]: /:id - here

File: src/routes/wellnessRoutes.ts [hunk 2]
```diff
@@ -7,6 +7,6 @@ const router = Router()
 router.route('/').get(auth, getWellness)
 router.route('/').post(auth, addWellness)
 router.route('/disable/:wellnessId').patch(auth, disableWellness)
-router.route('/update').patch(auth, updateWellness)
```
Comment (jonathans199) [inline src/routes/wellnessRoutes.ts:10]: this one should be `/update/:id`

File: src/routes/wellnessRoutes.ts [hunk 3]
```diff
@@ -7,6 +7,6 @@ const router = Router()
...
+router.route('/:wellnessId').patch(auth, updateWellness)
```
Comment (jonathans199) [inline src/routes/wellnessRoutes.ts]: this one also should be just `/:id`

File: src/routes/wellnessRoutes.ts [hunk 4]
```diff
@@ -7,6 +7,6 @@ const router = Router()
 router.route('/').get(auth, getWellness)
 router.route('/').post(auth, addWellness)
 router.route('/disable/:wellnessId').patch(auth, disableWellness)
```
Comment (jonathans199) [inline src/routes/wellnessRoutes.ts]: this one should be `/disable/:id`

**Review by jonathans199 — CHANGES_REQUESTED** [review]

File: src/routes/newsRoutes.ts (hunk 1 above)
Comment (ralphZero) [reply src/routes/newsRoutes.ts]: this one is outdated -
fixed in main --> https://github.com/bocacode/bocacode-intranet-api-typescript/blob/3157106c31086d249e09ad297f621d8b0768d270/src/routes/newsRoutes.ts#L12

**Review by jonathans199 — APPROVED** [review]

**Pull request merged**